packages/google_mlkit_text_recognition/ios/Classes/GoogleMlKitTextRecognitionPlugin.m

```diff
@@ -157,7 +163,9 @@ - (void)addData:(NSMutableDictionary *)addTo
    cornerPoints:(NSArray<NSValue *> *)cornerPoints
           frame:(CGRect)frame
       languages:(NSArray<MLKTextRecognizedLanguage *> *)languages
-           text:(NSString *)text {
+           text:(NSString *)text
```

Comment:
you are missing to add `confidence` and `angle` to the dictionary inside this method. Look at the end of that method and add it in `addEntriesFromDictionary`